TM1py/Services/CubeService.py

```diff
@@ -189,6 +189,40 @@ def get_dimension_names(self, cube_name: str, skip_sandbox_dimension: bool = Tru
             return dimension_names[1:]
         return dimension_names
 
+    def get_names_with_dimension(self, dimension_name: str, model_cubes_only: bool = False,
```

Comment:
Does `get_cube_names_with_dimension` sound better?

Reply:
I originally considered that, but felt the function name was too long and that the "cubes" was implied since it's in the cube service